Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -384,80 +173,85 @@ void Server::start()
     
     for (size_t i = 0; i < threadsCount; ++i)
     {
-        mThreads.emplace_back(std::thread([this]() { mContext.run(); }));
+        _threads.emplace_back(std::thread([this]() { _context.run(); }));
     }
-    mPostgreManager->handleRequests();
+    _postgreManager->handleRequests();
 
-    Base::Logger::FileLogger::getInstance().log("[SERVER] Started!", Base::Logger::LogLevel::INFO);
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "[SERVER] Started!", Base::Logger::LogLevel::INFO
+    );
 }
 
 void Server::stop()
 {
-    mContext.stop();
+    _context.stop();
 
-    mPostgreManager->stopHandler();
+    _postgreManager->stopHandler();
 
-    for (std::thread& thread : mThreads)
+    for (std::thread& thread : _threads)
     {
         if (thread.joinable())
         {
             thread.join();
         }
     }
 
-    mThreads.clear();
-    Base::Logger::FileLogger::getInstance().log("[SERVER] Stopped!", Base::Logger::LogLevel::INFO);
+    _threads.clear();
+
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "[SERVER] Stopped!", Base::Logger::LogLevel::INFO
+    );
 }
 
 void Server::waitForClientConnection()
 {
-    mAcceptor.async_accept([this](std::error_code error, asio::ip::tcp::socket socket) {
-        if (!error)
-        {
-            std::ostringstream out;
-            out << socket.remote_endpoint();
-            Base::Logger::FileLogger::getInstance().log
-            (
-                "[SERVER] New Connection: " + out.str(), 
-                Base::Logger::LogLevel::INFO
-            );
+    _acceptor.async_accept([this](std::error_code error, asio::ip::tcp::socket socket) { acceptingClientConnection(error,socket); });
```

Comment:
need investigation

Reply:
> 
Below there is a method that gets the parameters for the lambda expression to work correctly. 
It was made in order to reduce the code in the block and sonar would not be perturbed.